### ISSUE TITLE
Bug: `no-console` bad suggestion message with non-literal method names

### ISSUE THREAD

**Issue by fasttime** [eslint/eslint#19477]
### Environment

Node version: v23.9.0
npm version: v10.9.2
Local ESLint version: v9.21.0 (Currently used)
Global ESLint version: v9.21.0
Operating System: darwin 24.3.0

### What parser are you using?

Default (Espree)

### What did you do?

```js
/* eslint no-console: "error" */

console[logLevel]("problem");
console[42]("problem");
```

### What did you expect to happen?

The rule should produce suggestion messages that are not confusing.

### What actually happened?

I am getting suggestion messages that either don't match the source code or contain a syntax error.

> Remove the console.null().
> Remove the console.42().

### Link to Minimal Reproducible Example

https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLWNvbnNvbGU6IFwiZXJyb3JcIiAqL1xuXG5jb25zb2xlW2xvZ0xldmVsXShcInByb2JsZW1cIik7XG5jb25zb2xlWzQyXShcInByb2JsZW1cIik7XG4iLCJvcHRpb25zIjp7InJ1bGVzIjp7fSwibGFuZ3VhZ2VPcHRpb25zIjp7InNvdXJjZVR5cGUiOiJtb2R1bGUiLCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e319fX19

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

For computed method names an option would be to replicate the source code in the suggestion message, i.e.:

> Remove the console[logLevel].
> Remove the console[42].

Alternatively, there could be a generic message that doesn't include the method name.

> Remove the console method call.
